### ISSUE TITLE
Add tooltip for slider between variables' list and data frame

### ISSUE THREAD

**Issue by rodrigosf672** [posit-dev/positron#9674]
<img width="923" height="362" alt="Image" src="https://github.com/user-attachments/assets/21de0b62-57d7-46db-b0a2-d3a339555307" />

#### Positron and OS details:
```
Positron Version: 2025.10.0 build 156
Code - OSS Version: 1.103.0
Commit: 74261231013aba3599280cc235e255d77c7aa12c
Date: 2025-09-30T07:11:44.047Z
Electron: 37.2.3
Chromium: 138.0.7204.100
Node.js: 22.17.0
V8: 13.8.500258-electron.0
OS: Darwin arm64 25.0.0
```

## Describe the issue:
Add tooltip for slider between variables' list and data frame

## Steps to reproduce the issue:

1. Open data frame or `.csv` file in order to view the list of variables on the left and the data frame on right. Notice that there is a left/right-pointing arrow in the middle that allows for the variables list to be collapsed/displayed. 
2. Hover over the arrow and notice that there is no tooltip.

## Expected or desired behavior:
Tooltip should be present.
